jaeger-extension-native/src/main/java/io/ballerina/observe/trace/jaeger/JaegerTracerProvider.java

```diff
@@ -46,34 +52,56 @@ public void init() {    // Do Nothing
     }
 
     public static void initializeConfigurations(BString agentHostname, int agentPort, BString samplerType,
-                                                  BDecimal samplerParam, int reporterFlushInterval,
-                                                  int reporterBufferSize) {
-        // Create Sampler Configuration
-        samplerConfiguration = new Configuration.SamplerConfiguration()
-                .withType(samplerType.getValue())
-                .withParam(samplerParam.value());
-
-        // Create Sender Configuration
-        Configuration.SenderConfiguration senderConfiguration = new Configuration.SenderConfiguration()
-                .withAgentHost(agentHostname.getValue())
-                .withAgentPort(agentPort);
+                                                BDecimal samplerParam, int reporterFlushInterval,
+                                                int reporterBufferSize) {
+
         String reporterEndpoint = agentHostname + ":" + agentPort;
+        if (!reporterEndpoint.startsWith("http")) {
```

Comment:
Shouldn't this be ?
```suggestion
        if (!reporterEndpoint.startsWith("http://")) {
```

Reply:
Removed this section with recent changes.